Documentation/gettingstarted/dns.rst

```diff
@@ -0,0 +1,203 @@
+
+Let's use the above example and allow ``mediabot`` pods to access any twitter sub-domain i.e. allow a pattern ``*.twitter.com``. We will achieve this by changing the ``toFQDN`` rule to use ``matchPattern`` instead of ``matchName``.
+
+.. literalinclude:: ../../examples/minikube/dns-pattern.yaml
+
+.. parsed-literal::
+
+  $ kubectl apply -f \ |SCM_WEB|\/examples/minikube/dns-pattern.yaml
+
+Test that ``mediabot`` now has access to multiple twitter services for which the DNS matches the pattern ``*.twitter.com``. Important to note and test that this doesn't allow access to ``twitter.com`` because the ``*.`` in the pattern requires one subdomain to be present in the DNS name. You can simply add more ``matchName`` and ``matchPattern`` clauses to extend the access. 
+(`Learn more <http://docs.cilium.io/en/latest/policy/language/?highlight=DNS#dns-based>`_ about specifying DNS rules using patterns and names.)
+
+.. parsed-literal:: 
+  
+   $ kubectl exec -it mediabot -- curl -sL https://help.twitter.com
+   ...
+
+   $ kubectl exec -it mediabot -- curl -sL https://about.twitter.com
+   ... 
+
+   $ kubectl exec -it mediabot -- curl -sL https://twitter.com
+   ^C 
+
+
+Step 7: Combining DNS and Port Rules
+====================================
+
+The DNS-based access can be restricted to a specific port by adding an L4 rules section. Continuing with the example, we will restrict access to twitter services to port ``443``. 
+
+.. literalinclude:: ../../examples/minikube/dns-port.yaml
+
+.. parsed-literal::
+
+  $ kubectl apply -f \ |SCM_WEB|\/examples/minikube/dns-port.yaml
+
+Testing, the access to ``https://help.twitter.com`` will succeed since it access on port 443 but access to ``http://help.twitter.com`` will be blocked since it goes to port 80.
+
+.. parsed-literal:: 
+  
+   $ kubectl exec -it mediabot -- curl -sL https://help.twitter.com
+   ...
+
+   $ kubectl exec -it mediabot -- curl -sL http://help.twitter.com
+   ^C  
+
+Step 8: Combining DNS, Port and API-aware Rules
+===============================================
+
+DNS and port can be combine with API-aware (L7) rules to further restrict access. This is particularly useful for internal shared service such as Cassandra, Kafka, Memcache, Redis, or HTTP-based services. The below example allows ``spaceship`` pods to access an external service, `starwars.covalent.link` but it restricts the access to a specific port, ``80``, and to a specific action and resource, ``HTTP GET /naboo/landing-request``. 
+
```

Comment:
The yaml isn't included here so it shows up, was that intended? It's the only step without it, I think.

Reply:
I'll fix that